webapp/src/main/java/com/box/l10n/mojito/rest/repository/BranchStatisticSpecification.java

```diff
@@ -14,12 +14,22 @@
 import javax.persistence.criteria.JoinType;
 import javax.persistence.criteria.Predicate;
 import javax.persistence.criteria.Root;
+import java.util.List;
 
 /**
  * @author jeanaurambault
  */
 public class BranchStatisticSpecification {
 
+    public static SingleParamSpecification<BranchStatistic> branchStatisticIdIn(List<Long> branchStatisticIds) {
```

Comment:
is that still used?

Reply:
Not anymore, forgot to clean up, will remove.